ISISComputingGroup/EPICS-ioc
# Update DB files to set interesting calc records to be readonly

### Description of work
This PR adds all `calc` PVs that are marked as interesting to the access security group `readonly` as they should not be directly set. The other PRs in this issue also add field to db files in the support repos and a check in the `DbUnitChecker`

Other relevant PRs:
 - ISISComputingGroup/EPICS-DbUnitChecker#7
 - ISISComputingGroup/EPICS-SKFChopper#6

### To test
 - Pick one of the PVs that has the new security group field.
 - Try and do a caput to that record. You should not be able to set the value.
 - Try setting it through the IBEX gui python console. You should not be able to set it and you should get a sensible error message.
 - Remove one of the ASG fields and try running the `DbUnitChecker`. It should flag an error if the record is interesting and not readonly.

Issue is: ISISComputingGroup/IBEX#2343

### Acceptance criteria
 - Interesting `calc`, `transform` and `calcout` PVs are now set to be readonly.
 - The `DbUnitChecker` check works.


---

#### Code Review


### Functional Tests


### Final steps
